ui/components/SpacesSwitcher/SpaceSwitcher.js

```diff
@@ -30,6 +30,8 @@ import {
   useUpdateSelectedOrganizationMutation,
 } from '@/rtk-query/user';
 import { MobileOrgWksSwither } from './MobileViewSwitcher';
+import WorkspaceModal from './WorkspaceModal';
+import { WorkspaceModalContext } from '@/utils/context/WorkspaceModalContextProvider';
```

Comment:
@FaheemOnHub what does this context provider do , is it being use somewhere else too?